assets/sass/protocol/components/_button.scss

```diff
@@ -139,8 +138,7 @@ a.mzp-c-button {
 // Primary
 
 /* stylelint-disable-next-line no-duplicate-selectors */
```

Comment:
```suggestion
```

Reply:
We do still have a duplicate selector here (there's no `mzp-t-primary` theme class to distinguish this selector from the other default button styles), but this is the only place we need the disable comment now 

<img width="987" alt="duplicate-selector" src="https://github.com/user-attachments/assets/e1f80b00-89fe-4b6e-a217-4debd5c5e643" />